src/NHSD.BuyingCatalogue.Organisations.Api/Services/EmailAddress.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using MimeKit;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.Services
+{
+    /// <summary>
+    /// An e-mail address.
+    /// </summary>
+    internal sealed class EmailAddress
+    {
+        private string address;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EmailAddress"/> class.
+        /// </summary>
+        /// <remarks>Required by <see cref="System.Text.Json"/>.</remarks>
+        public EmailAddress()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EmailAddress"/> class
+        /// with the given name and address.
+        /// </summary>
+        /// <param name="name">An optional </param>
+        /// <param name="address"></param>
+        internal EmailAddress(string name, string address)
+        {
+            Name = name;
+            Address = address;
+        }
+
+        /// <summary>
+        /// Gets or sets the display name of the address.
+        /// </summary>
+        /// <remarks>An optional display name, for example
+        /// Buying Catalogue Team.</remarks>
+        public string Name { get; set; }
```

Comment:
Could this be named DisplayName? Initially I thought this was the name of the user.

Reply:
In our case it will be the name of the user for recipients (or the name of a team in the case of our sender) or did you think it was the user's username?

I'm happy to rename this as suggested, regardless.